src/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import "./PokemonCard.css";
+
+function PokemonCard({endPoint}) {
+    const [cardData, setCardData] = useState("");
+
+    useEffect(() => {
+        async function fetchCard() {
+            try {
+                const {data} = await axios.get(endPoint)
+                setCardData(data)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        void fetchCard()
+    }, [endPoint]);
+
+
+    return (
+        <>
+            {cardData !== "" && <section className="border">
```

Comment:
Als je het op zo'n manier doet dan heb je wat minder nesting en leest het dus wat makkelijker
```suggestion
    if (!cardData) return null;
    
    return (
```